core/server/services/email-analytics/lib/event-processor.js

```diff
@@ -88,6 +88,17 @@ class GhostEventProcessor extends EventProcessor {
                 opened_at: this.db.knex.raw('COALESCE(opened_at, ?)', [moment.utc(event.timestamp).format('YYYY-MM-DD HH:mm:ss')])
             });
 
+        const {value: timezone} = await this.db.knex('settings').first('value').where('key', 'timezone');
```

Comment:
I've added a fallback ✅ 

Could you share in which case it would happen?
I was assuming that a DB without a `timezone` setting could only be the result of a migration, in which case the person writing the migration would make sure nothing would break in the whole Ghost codebase.

Reply:
It _should_ always be present, but I think this is one of those cases where it's better to be defensive, especially as we're not going through the settings cache for this.